### ISSUE TITLE
Build Fails After Installing Plugin

### ISSUE THREAD

**Issue by thandaanda** [chemerisuk/cordova-plugin-firebase-dynamiclinks#12]
Hi @chemerisuk,
I am getting google-services.json file not found error while building for Ionic 3.

`Could not find google-services.json while looking in [src/nullnull/debug, src/debug/nullnull, src/nullnull, src/debug, src/nullnullDebug]`

`> File google-services.json is missing. The Google Services Plugin cannot function without it.
   Searched Location:
  /Users/user/Documents/Git/ionic/platforms/android/app/src/nullnull/debug/google-services.json
  /Users/user/Documents/Git/ionic/platforms/android/app/src/debug/nullnull/google-services.json
  /Users/user/Documents/Git/ionic/platforms/android/app/src/nullnull/google-services.json
  /Users/user/Documents/Git/ionic/platforms/android/app/src/debug/google-services.json
  /Users/user/Documents/Git/ionic/platforms/android/app/src/nullnullDebug/google-services.json
  /Users/user/Documents/Git/ionic/platforms/android/app/google-services.json`

**Comment by fcarreno** [comment]
I have the same issue...

**Comment by fcarreno** [comment]
UPDATE: it seemed to be related with different versions of google play/firebase libraries being used by different plugins. Using the --variable FIREBASE_DYNAMIC_LINKS_VERSION parameter during plugin installation made the trick (to align it to the versions used by other plugins).

So my project.properties file ended up like this (among other lines):

....
cordova.system.library.3=com.google.firebase:firebase-core:12.0.0
cordova.system.library.4=com.google.firebase:firebase-messaging:12.0.0
cordova.system.library.6=com.google.android.gms:play-services-auth:12.0.0
cordova.system.library.7=com.google.android.gms:play-services-identity:12.0.0
cordova.system.library.8=com.google.firebase:firebase-dynamic-links:12.0.0

Using --variable FIREBASE_DYNAMIC_LINKS_VERSION="12.0.0" from the cordova plugin add command.
I tested a link real quick, and it worked just fine, but not sure if all dynamic link features will (e.g.: survive after installation) since the plugin by default seems to require version 16.0.+

At least with this I was able to get a successful build...
